pos_customer_required/migrations/9.0.2.0/post-migration.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+BOOLEAN_TO_SELECTION = {
+    'false': 'no',
+    'true': 'order',
+}
+
+
+def migrate_require_customer(cr):
+    for old, new in BOOLEAN_TO_SELECTION.iteritems():
```

Comment:
You can use openupgradelib `map_values` method: https://github.com/OCA/openupgradelib/blob/4bb94fec0e3571cbc530e09b9c5b7903ece35f00/openupgradelib/openupgrade.py#L801